src/pages/journal-vouchers/forms/JournalVoucherForm.js

```diff
@@ -64,51 +62,74 @@ export default function JournalVoucherForm({ labels, access, recordId, general =
         })
 
         if (!recordId) {
-          toast.success('Record Added Successfully')
-
-          const res = await getRequest({
-            extension: GeneralLedgerRepository.JournalVoucher.get,
-            parameters: `_recordId=${response.recordId}`
-          })
-
-          formik.setValues(res.record)
-        } else toast.success('Record Edited Successfully')
+          toast.success(platformLabels.Added)
+          getData(response.recordId)
+        } else toast.success(platformLabels.Edited)
 
         invalidate()
       } catch (error) {}
     }
   })
+
   const editMode = !!formik.values.recordId || !!recordId
+  const isRaw = formik.values.status == 1
```

Comment:
`const isRaw = formik.values.status == 1
const isPosted = formik.values.status === 3`
u have 2 const where it's value is based on formik.values.status can we remove one of them to prevent duplicates?
for example readonly fields can be readOnly={editMode || isPosted}

check if it's possible and test it